src/System.Collections.NonGeneric/tests/Performance/Perf.HashTable.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using Microsoft.Xunit.Performance;
+using System.Collections.Generic;
+
+namespace System.Collections.Tests
+{
+    public class Perf_HashTable : PerfTestBase
+    {
+        private IEnumerable<Hashtable> TestHashtables()
+        {
+            yield return CreateHashtable(100);
+            yield return CreateHashtable(1000);
+        }
+
+        private Hashtable CreateHashtable(int size)
+        {
+            Hashtable ht = new Hashtable();
+            for (int i = 0; i < size; i++)
+                ht.Add(CreateString(50), CreateString(50));
+            return ht;
+        }
+
+        [Benchmark]
+        public void ctor()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    new Hashtable();
```

Comment:
This is the case for nearly every microbenchmark that I've written and is a currently a topic of discussion over at https://github.com/Microsoft/xunit-performance/issues/62.

you make a good point about the disposer that I hadn't considered, though. I don't really want to measure disposal of the Hashtable within ```iteration.StartMeasurement()``` (since I'm testing the constructor here), so I should retain a local reference until outside of that using to avoid it being disposed too early.